.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  push:
+    branches:
+      - master
+
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
I'm not certain how `./gradlew -i publishToMavenCentral` is determining the `version` during a build and publish. If it's relying on git tags (which is our standard method of determining versions), then in Github Actions we need to make sure we include `fetch-depth: 0` during the checkout stage so that all the git history and tags are available to CI.

See the change I made here: https://github.com/brightsparklabs/appcli/commit/32e37dd7b34d98fbb03e9351f3f90d7db5f9b9f1